packages/deploy/deploy/300_catalyst/300_catalyst_mint.ts

```diff
@@ -0,0 +1,43 @@
+import {DeployFunction} from 'hardhat-deploy/types';
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const {deployments, getNamedAccounts} = hre;
+  const {execute, catchUnknownSigner, log} = deployments;
+  const {catalystMinter} = await getNamedAccounts();
+
+  // TODO Clarify whether the below is the correct contract name
+  const GiveawayContract = await deployments.get('MultiGiveawayV1');
```

Comment:
@wojciech-turek contract name is `MultiGiveaway` (Mumbai classic giveaway at https://mumbai.polygonscan.com/address/0xfce84d07909489508c5b293a850af15fb7147bc6#code)